samples/nrf9160/modem_shell/README.rst

```diff
@@ -720,6 +784,18 @@ The LEDs have the following functions:
 LED 3 (nRF9160 DK)/Blue LED (Thingy:91):
    Indicates the LTE registration status.
 
+Power measurements
+==================
+
+Power measurements can be conducted using `Power Profiler Kit II (PPK2)`_.
+To achieve proper power measurement numbers, UART must be disabled.
+This can be done in modem shell with one of the following actions:
+
+  * Use MoSh command ``uart`` to disable uarts as mentioned in :ref:`uart_command`
```

Comment:
```suggestion
To achieve proper power measurement numbers, UARTs must be disabled.
This can be done in MoSh with one of the following actions:

  * Use MoSh command ``uart`` to disable UARTs as mentioned in :ref:`uart_command`
```